interp/constrintern.ml

```diff
@@ -359,13 +359,14 @@ let reset_hidden_inductive_implicit_test env =
          | (Inductive (params,_),b,c,d) -> (Inductive (params,false),b,c,d)
          | x -> x) env.impls }
 
-let check_hidden_implicit_parameters id impls =
+let check_hidden_implicit_parameters loc id impls =
```

Comment:
Thanks Hugo, #402 makes it globally consistent, I am assuming we would ultimately like that PR to go in.